omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
+    transfer: "ln_s"
+    path: "my-files.txt"
+
+Assuming that my-files.txt is a list of files such as ::
+
+    fileA
+    fileB
+    fileC
+
+this is equivalent to: ::
+
+    $ bin/omero import -k --transfer=ln_s fileA fileB fileC
+
+Bulk-only options
+-----------------
+
+Path
+^^^^
+
+The ``path`` key specifies a file from which *each individual line*
+will be processed as a separate import. In the simplest case, a single
+file is placed per line as above. For more complex usages, ``path``
+can point to a tab-separated value (TSV) file where each field will
+be interpreted based on ``columns``.
```

Comment:
Is `columns` required if a TSV file is passed or is only the first column considered?

Reply:
My guess is it will assume the filename has a tab in it.

Reply:
I made an example in basel to import folder only
but I used csv.
cvs might be more natural in some cases

Reply:
I don't understand what you mean, @jburel.

Reply:
if you copy one of our examples, the tab usually disappears when the content is copied in a text editor, leading to some errors when trying to run the command. This can be tricky to understand/debug.

Using csv file avoids that issue